sample_app/app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
```

Comment:
Hàm này thực hiện những chưc năng gì?

Reply:
Lưu lại các thuộc tính của user vừa log in (session vẫn đang lưu )